src/ClimaLSM.jl

```diff
@@ -74,7 +74,10 @@ function Domains.coordinates(model::AbstractLandModel)
     return coords
 end
 
-function initialize_prognostic(model::AbstractLandModel{FT}, coords) where {FT}
+function initialize_prognostic(
+    model::AbstractLandModel{FT},
+    coords::Union{ClimaCore.Fields.Field, ClimaCore.Fields.FieldVector},
```

Comment:
ditto here - do we use this with a FieldVector as input?

Reply:
`docs/tutorials/Usage/LSM_single_column_tutorial.jl` calls this function with parameter types `::AbstractLandModel, ::FieldVector`, using a `LandHydrologyModel`.  If I change this line to just `coords::ClimaCore.Fields.Field`, we get a method error when running that file

Reply:
oh ok! youre right! all good, sorry about that.